packages/eslint-config-mocha/index.js

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Extraneous empty line

Reply:
Actually, maybe adding `'no-multiple-empty-lines': ['error', { max: 1, maxEOF: 1, maxBOF: 0 }],` to the base config can actually stop all these types of issues going forward!

Reply:
Just tested and it looks like it actually needs to have `maxEOF: 0` otherwise it allows two empty lines at the end of the file. Kind of a strange setting here, but I tested it out.

Reply:
Yeah please tweak it as necessary!